bentoml/_internal/bento/build_config.py

```diff
@@ -488,6 +484,44 @@ def write_to_bento(self, bento_fs: "FS", build_ctx: str) -> None:
                     "Falling back to using user-provided package requirement specifier, equivalent to `lock_packages=False`"
                 )
 
+        with bento_fs.open(
+            fs.path.join(py_folder, "install.sh"), "w"
+        ) as install_script:
+            args = " ".join(map(quote, pip_args)) if pip_args else ""
+            install_python_packages = (
+                """\
+#!/usr/bin/env bash
+
+# https://stackoverflow.com/a/246128/8643197
+BASEDIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]:-$0}"; )" &> /dev/null && pwd 2> /dev/null; )"
```

Comment:
this requires calling the script from the Bento directory?

Reply:
BASEDIR will be the `env/python` directory?

Reply:
You can call this script anywhere, and `BASEDIR` will always be `env/python`